cmd/gh-slack/cmd/root.go

```diff
@@ -10,6 +10,10 @@ var rootCmd = &cobra.Command{
 	Use:   "gh-slack [command]",
 	Short: "Command line tool for interacting with Slack through gh cli",
 	Long:  `A command line tool for interacting with Slack through the gh cli.`,
+	Example: `  gh-slack -i <issue-url> <slack-permalink>  # defaults to read command
+  gh-slack read <slack-permalink>
+  gh-slack read -i <issue-url> <slack-permalink>
+  gh-slack send -m <message> -c <channel-id> -t <team-name>`,
```

Comment:
nit: A placeholder like `<message-text>` may be clearer than just `<message>`. In the future, if / when we support attachments / JSON payloads, we could revisit the naming of the placeholder.